frontend/src/DropdownMenu/DropdownMenu.css

```diff
@@ -0,0 +1,5 @@
+.DropdownMenu {
+  position: absolute;
+  top: 60px;
```

Comment:
This seems to work well! Now let's use [makeStyles](https://material-ui.com/styles/basics/#hook-api) and Material UI's spacing components in DropdownMenu.tsx with whole-number margins (see [here](https://material-ui.com/system/spacing/)). This allows for margins to be parametrized and changed globally later.